src/libraries/tests.proj

```diff
@@ -492,6 +492,8 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Net.Mail\tests\Functional\System.Net.Mail.Functional.Tests.csproj"
                       Condition="'$(TargetOS)' == 'windows'"/>
 
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Data.DataSetExtensions\tests\System.Data.DataSetExtensions.Tests.csproj" />
```

Comment:
This is the place where we dump projects that were not triaged.

The projects that were looked at and have an active bug are at the top of the ItemGroup, with a link to the issue.

It would be better if we could do this with an ActiveIssue on a specific test, but I don't know how many of the tests are failing here.

Reply:
I think this may be fixable. Likely just add some rd.xml. I just did not have time to do that.

Reply:
The failure mode is suspicious, like I wrote in https://github.com/dotnet/runtime/issues/79513#issuecomment-1345748617. This would not be RD.XML. The library is supposed to be AOT-compatible - we did the work in the summer.

I cannot repro this locally:

```
C:\git\runtime\artifacts\bin\System.Data.DataSetExtensions.Tests\Release\net8.0\publish>System.Data.DataSetExtensions.Tests.exe -notrait category=failing
Running assembly:System.Data.DataSetExtensions.Tests, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
Finished System.Data.DataSetExtensions.Tests, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51

Tests run: 103, Errors: 0, Failures: 0, Skipped: 0. Time: 0.0294375s
```

Reply:
I dug out the failure from pipelines (force pushing removed all the Github UI to access that). I wiped my artifacts and going to try to repro this locally with a clean slate. If we're getting the stack the CI is getting, something is seriously wrong.